lib/susedistribution.pm

```diff
@@ -198,6 +198,7 @@ sub become_root {
     my ($self) = @_;
 
     $self->script_sudo('bash', 0);
+    wait_still_screen;
```

Comment:
this won't help. We don't see the 'root' in serial0.txt, so it's just not there - no matter how long you wait for it.